src/app/config.py

```diff
@@ -30,7 +30,8 @@
     raise ValueError(
         "Missing Credentials. Please set 'SUPERUSER_LOGIN' and 'SUPERUSER_PWD' in your environment variables")
 
-BUCKET_NAME: str = "mypyroneartest"
+QARNOT_TOKEN: str = os.getenv("QARNOT_TOKEN")
```

Comment:
Don't know how QARNOT works but wouldn't it be better to have login/password?
The token has probably an expiration date; plus it may be more cumbersome to retrieve the Token by hand on your machine.

How do you put the token in the environment variable? It must be set in the docker-compose to be able to use it in the docker container.

Reply:
I am mot sure that we have the possibility to do that with QARNOT; all examples I have seen work with token; I think the token does not expire but I could be regenerated on demand.

Yes, the token must be set in the env variable on Heroku. I didn't put in docker-compose to keep it private and since we don't need it on test/dev side. I am not really testing the buckets in the test; all methods are patched

Reply:
This token is not supposed to expire, but on the contrary for security reason, it's much much better to use a token authentication. If the token gets compromised, we can very easily disable it. But if the CSP creds get compromised, we would be in big trouble.

I agree for deployment on Heroku, and for local usage, perhaps we should be exploring making an `.env` file at some point :thinking: